example_school_library_decorator/book.rb

```diff
@@ -1,7 +1,24 @@
+# rubocop:disable all
+
 class Book
+  @@books = []
```

Comment:
- [ ] I noticed that you have this class variable in your book class, your book class is responsible for creating a` book` instance and not storing books. Kindly create this variable in your` Bookstore` || `App` class that is responsible for storing the books that you create.